backends/arm/operators/op_rescale.py

```diff
@@ -48,7 +47,10 @@ def define_node(
                 f"If output dtype is not int8, output_zp must be 0. Got {output_dtype=}, {output_zp=}"
             )
 
-        scale_width = 32 if output_dtype == torch.int32 else 16
+        # scale32 gives higher accuracy but for a higher HW cost.
+        # For now, always go for scale32.
+        scale_32 = True
```

Comment:
- how much more to use wider scale?
- why do we always want to do mul_t = i32_t? can we check `multiplier`, `shift` and input value dtype to decide better?
- do we care to specify rounding mode if scale_32 = True?

Reply:
> We do have an internal ticket for investigating this.
Thanks, just curious. No need to spend more efforts tbh.

> but maybe not in this PR.
sounds OK.

> But maybe I am wrong?
let's keep double=False unless we need to.